Python/import.c

```diff
@@ -396,15 +396,18 @@ static const char * const sys_files[] = {
 void
 PyImport_Cleanup(void)
 {
+    assert(!PyErr_Occurred());
```

Comment:
The contract of this function is that the caller must not hold an exception. If the assertion fails, I would prefer to fix the caller rather than working around the bug using PyErr_WriteUnraisable().

The point is not only to catch bugs in the current code, but also ease debug when the callers code will be modified. Python shutdown process is very fragile and error prone.